test/unitTests/core/test-seat.js

```diff
@@ -0,0 +1,135 @@
+import { test } from 'tape-promise/tape';
+import harden from '@agoric/harden';
+
+import { makeSeatMint } from '../../../core/seatMint';
+import { makeMint } from '../../../core/issuers';
+import { insist } from '../../../util/insist';
+
+/*
+ * A seat quantity may look like:
+ *
+ * {
+ *   id: {},
+ *   offerToBeMade: [rule1, rule2],
+ * }
+ *
+ * or:
+ *
+ * {
+ *   id: {},
+ *   offerMade: [rule1, rule2],
+ * }
+ *
+ */
+
+const allTrue = (prev, curr) => prev && curr;
```

Comment:
call it bothTrue?

Reply:
Do you think that's clearer as the parameter to `array.reduce`? To me it's clearer that I'm checking that everything in the array is true when it's called `allTrue` instead of `bothTrue`, even though of course reduce is comparing two items: a new item to the reduction of everything before it.

Reply:
`allTrue()` makes it seem like a predicate I could pass more than two parameters to. This would result in it ignoring all but the first two, which could lead to some surprising results. If it could only be called in `reduce()`, this wouldn't be an issue, but I don't see anything that prevents other uses.

I do agree that it reads better in `reduce()` calls.

Reply:
Ok, that makes sense to me. I just changed `allTrue` to `bothTrue`